sets.h

```diff
@@ -0,0 +1,51 @@
+/*
+    Create sets.
+
+    Copyright (C) 2016-2017 Bijan Kazemi <bkazemi@users.sf.net>
+
+    This file is part of libscanmem.
+
+    This library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this library.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef SETS_H
+#define SETS_H
+
+#include <stdlib.h>
+
+struct set {
+    size_t  sz;  /* size of set (used) */
```

Comment:
Can you please call the set member `size`?
If you prefer the abbreviate `sz` for the local variables of the set parser, that's fine, but I'd like a readable name in the rest of the program.

Reply:
Changed.